.github/workflows/schedule.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedules:
+  - cron: 0 9 * * *
+
+name: Update data
```

Comment:
And this one can maybe be:
```
name: Update API History Data
```

this is a total nit, but it looks a bit better in the new UI :)